### PR TITLE
Add pause button

### PR DESCRIPTION
**Pull request by ViryBe**
Just add a button in order to pause current music.
Pushing a second time 'pause' re-start the music from where it was paused.

### CODE REVIEW

**Review by jboynyc — COMMENTED** [review]
Cool, thanks so much for submitting this! I'll review and merge when I get a chance.

Would you be willing to contribute an updated screenshot for the readme as well?

**Comment by ViryBe** [comment]
Actually it doesn't change anything on the screenshot. I only change the behavior of the play button.

**Comment by jboynyc** [comment]
Awesome, very elegant.

File: app.py [hunk 1]
```diff
@@ -87,7 +88,7 @@ def get_status():
         play = out[0].split()[1]
         if play == 'playing':
             r['playing'] = True
-        elif play == 'stopped':
```
Comment (FriedrichFroebel) [inline app.py]: This change breaks the displayed title when stopping the playback as `response.playing` is missing from the dictionary. Appending to the (not existing) variable `msg` inside the JavaScript code therefore results in an artist which starts with `undefined`.

**Comment by FriedrichFroebel** [comment]
https://github.com/jboynyc/cmus_app/pull/2/files#diff-3f41e546893dc64b71aaacad12cad815L90 breaks the displayed title when stopping the playback as `response.playing` is missing from the dictionary. Appending to the (not existing) variable `msg` inside the JavaScript code therefore results in an artist which starts with `undefined`.

**Comment by jboynyc** [comment]
I'm not using `cmus` anymore, so I can't test this. I'll close the PR. Let me know if you'd like me to link to your fork from the README.

**Pull request closed without merging**